geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/iterators/Z2Iterator.scala

```diff
@@ -124,20 +112,22 @@ class Z2Iterator extends SortedKeyValueIterator[Key, Value] {
 object Z2Iterator {
 
   val ZKeyXY = "zxy"
-  val PointsKey = "points"
-  val TableSharingKey = "table-sharing"
+  val ZOffsetKey = "zo"
+  val ZLengthKey = "zl"
 
-  val RangeSeparator = ":"
+  private val RangeSeparator = ":"
+  private val TermSeparator  = ";"
 
-  def configure(bounds: Seq[(Double, Double, Double, Double)],
-                isPoints: Boolean,
-                tableSharing: Boolean,
-                priority: Int) = {
+  def configure(sft: SimpleFeatureType,
```

Comment:
figured it was better to do it now - the alternative is not merge this, or make some of the changes but not the `init` changes - however we observed that `init` was being called a lot more than we realized